hack/ci/run-dualstack-e2e-test.sh

```diff
@@ -74,6 +79,8 @@ export VSPHERE_PASSWORD="${VSPHERE_PASSWORD:-$(vault kv get -field=password dev/
 echodate "Successfully got secrets for dev from Vault"
 echodate "Running dualstack tests..."
 
-go_test dualstack_e2e -race -timeout 1h -tags dualstack -v ./pkg/test/dualstack/... -args --cni $CNI --provider $PROVIDER --os $OSNAMES
+export OSNAMES="${OSNAMES:-all}"
+
+go_test dualstack_e2e -race -timeout 90m -tags dualstack -v ./pkg/test/dualstack/... -args --cni $CNI --provider $PROVIDER --os $OSNAMES
```

Comment:
`90m` is way too long for the timeout?

Reply:
It is a long time. Unfortunately it is necessary because certain combinations take a long time to come up e.g. Equinix.

Reply:
Ah then it makes sense 👍